apps/cnp/plum-batch/plum-batch.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: plum-batch
+spec:
+  releaseName: plum-batch
+  values:
+    job:
+      image: https://hmctspublic.azurecr.io/hmcts/plum-batch:latest
```

Comment:
Is this just to test or long term? If long-term, can you update with relevant image tag & add image tag policy